annotations/src/main/java/org/robolectric/annotation/ConscryptMode.java

```diff
@@ -9,13 +9,13 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.PACKAGE, ElementType.TYPE, ElementType.METHOD})
-public @interface SecurityMode {
+public @interface ConscryptMode {
```

Comment:
Does `SecurityMode` exist at previous Robolectric release? If yes, we can't change it's name directly, because it will break user's previous tests that use `SecurityMode` when upgrading Robolectric to next release. If not, what about providing a proper name? The `ConscryptMode` is bound to implementation, but previous `SecurityMode` is more generic.

Reply:
Could you explain why you want to rename `SecurityMode` to `ConscryptMode`?

Reply:
SecurityMode is new, added a couple of weeks ago, so its safe to change.

Reply:
@utzcoz we renamed it because: instead of having either Conscrypt OR Bouncycastle, we are going to use both of them. 
- If ConscryptMode is ON, it will install Conscrypt and BouncyCastle.
- If ConscryptMode is OFF, it will not install Conscrypt but BouncyCastle will still be there.

Reply:
We need ON/OFF options for Conscrypt only, because BouncyCastle will be installed in both the conditions. So, it makes more sense to rename it.

Reply:
@Umesh-01 I got it. Thanks for explaining. Could you add more comments for SecurityProvider selection?

Reply:
done